doc/conf.py

```diff
@@ -27,7 +27,7 @@
 templates_path = ['_templates']
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
 
-intersphinx_mapping = {'py': ('https://docs.python.org/3.12', None)}
+intersphinx_mapping = {'py': ('https://docs.python.org/3.13', None)}
```

Comment:
Alternatively, this could link to the latest 3.x docs if pinning to a particular version isn't important:
```suggestion
intersphinx_mapping = {'py': ('https://docs.python.org/3', None)}
```

Reply:
I'm not a Sphinx expert, but that _sounds_ like it makes more sense to me, because then we don't have to remember to keep bumping this?

Reply:
I think it makes sense, unless we're worried about link rot between Python versions? Maybe @JelleZijlstra remembers if it was set up this way for a reason in #166.